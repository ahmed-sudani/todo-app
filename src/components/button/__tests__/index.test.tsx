import { describe, expect, it } from 'vitest'
import Button from '..'
import { render } from '@testing-library/preact'

describe('Button', () => {
  it('should render successfully', () => {
    const text = 'test'
    const { container } = render(<Button text={text} />)
    expect(container).toMatchSnapshot()
  })
  it('should display text successfully', () => {
    const text = 'test'
    const { container } = render(<Button text={text} />)
    expect(container.textContent).toEqual(text)
  })
})
