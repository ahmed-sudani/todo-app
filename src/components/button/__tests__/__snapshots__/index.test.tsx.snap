// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Button > should render successfully 1`] = `
<div>
  <button
    class="btn btn-rounded"
  >
    test
  </button>
</div>
`;
